### PR TITLE
Gregtech tools Fix

### PR DESCRIPTION
**Pull request by Minepolz320**
I will try to fix the tools
+Wire-Cutters Sound
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8480

### CODE REVIEW

**Comment by RinWate** [comment]
I think, it's very needful fix...

**Comment by Dream-Master** [comment]
@GTNewHorizons/developers @GTNewHorizons/core review please. Thanks

File: src/main/java/gregtech/api/items/GT_MetaGenerated_Tool.java [hunk 1]
```diff
@@ -463,18 +463,9 @@ public final boolean doDamage(ItemStack aStack, long aAmount) {
...
-        if (!isItemStackUsable(aStack))
-            return 0.0F;
-
+        if (!isItemStackUsable(aStack)) return 0.0F;
         IToolStats tStats = getToolStats(aStack);
-
-        if (tStats == null || Math.max(0, getHarvestLevel(aStack, "")) < aBlock.getHarvestLevel(aMetaData))
-            return 0.0F;
-
-        if (aBlock.getHarvestLevel(aMetaData) == 0 && !tStats.isMinableBlock(aBlock, (byte) aMetaData))
```
Comment (glowredman) [inline src/main/java/gregtech/api/items/GT_MetaGenerated_Tool.java:475]: Why is this part removed?

**Review by glowredman — APPROVED** [review]
Looks good as far as I can tell. @Minepolz320 I guess you tested your changes ingame?

**Comment by Minepolz320** [comment]
@glowredman seems fine if something go wrong message me i fix it
